### DISCUSSION TITLE
HwV380E11_WF3_IPC38_ETH

### DISCUSSION THREAD

**Discussion by SergeHendrickx** [Arkady23/V380-Web#38] [Show and tell]
Hi all,

Thanks for the great work on this project!
Spent a lot of time today trying to find out which HW id my camera mapped to, so I just wanted to share my findings.

In the app the firmware looks like this: **v380e11_wf3_ipc38_eth_20180628**

Log files:

> **[app_version]**
> name=AppV380E11_IPC38_ETH_V2.6.4.0
> date=20200429
> **[kernal_version]**
> name=KerV380E11_IPC38_ETH_V2.2.0
> date=20180628
> **[hardware_version]**
> name=HwV380E11_WF3_IPC38_ETH
> date=20180628
> **[base_version]**
> name=AppV380E11_IPC38_ETH_V2.6.4.0
> date=20200429

And eventually I was able to intercept the firmware request to the server:
### {"soft_id":**"V380E2_C2"**,"dev_id":__,"version":2060400,"timestamp":___}

Response on this was {"result":-1005,"interval":7200}

So it's obviously an already existing version of the web-app, but the mapping is far from clear so that's why I'm sharing this.

**Comment by Arkady23** [comment]
httpd_V380E2_C2.zip fit?
